Elle624/FitLit-Refactor
# Feature/user input modal

## Is this a fix or a feature?
feature

## What is the change?
There is now a button on the left side of the header. When clicked, it activates a modal pop-up with a form.
This allows user to enter sleep, hydration, and activity data. 
There is a submit button that is disabled until the user has completed each input.
On submit, the data is posted to the server. 

## What does it fix? 
N/A

## Where should the reviewer start?
_modal.scss: line 1
_text.scss: line 10
styles.scss: line 9
domUpdate.js: line 3
index.html: line 12
scripts.js: line 54, line 126

## How should this be tested?
npm start, open site, click "+" button in top left corner
enter data in form, click submit.
Check that new sleep data was posted in the last entry on: https://fe-apps.herokuapp.com/api/v1/fitlit/1908/sleep/sleepData

### Screenshot (optional)


@atownse
@elle624
@lo-la-do-li
